spec/controllers/locations_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe LocationsController do
+  render_views
+
+  describe '#index' do
+    let!(:location) { FactoryGirl.create(:location, uid: '25de9301-50b5-49ba-a5da-7f40a2fcfe29') }
+
+    it 'renders the location as JSON' do
```

Comment:
This test is actually testing the behaviour of the serializer. Perhaps it would be better to break that out into a specific spec, and just test here that the controller responds OK and with the JSON content-type? Per their respective responsibilities.